DDCore/include/DD4hep/ConditionAny.h

```diff
@@ -104,15 +106,26 @@ namespace dd4hep {
     /// Item part of the identifier
     itemkey_type item_key()  const;
 
+    /// Flag operations: Get condition flags
+    mask_type flags()  const;
+    /// Flag operations: Set a conditons flag
+    void setFlag(mask_type option);
+    /// Flag operations: UN-Set a conditons flag
+    void unFlag(mask_type option);
+    /// Flag operations: Test for a given a conditons flag
+    bool testFlag(mask_type option) const;
+
```

Comment:
For what I see in this MR, these functions are never used. Is this correct ? If yes, why to add them ?